routes/api.php

```diff
@@ -66,10 +62,9 @@
 Route::get('/profile/{address}',[ProfileController::class, 'getProfile']);
 Route::get('/protocols', [DataController::class, 'getProtocols']);
 Route::get('/circles', [CircleController::class, 'getCircles']);
-//// not used for now
-//Route::post('/circles', [CircleController::class, 'createCircle']);
-////
-
+Route::middleware(['hcaptcha-verify','verify-sign'])->group(function() {
+    Route::post('/create-circle', [CircleController::class, 'createCircle']);
```

Comment:
Typically in a RESTful API design the create for a resource `circle` would probably use a
POST to `/circles` to create a new circle.

Could we just use that RESTful pattern?


```
Common REST urls resemble the following:

    /static_root/
        GET - retrieve list of resources
            optionally include search params /static_root/?size=20&page=5
        POST - create resource
    /static_root/[id]/
        GET - retrieve one entity matching 'id'
        PUT - update entire entity
        PATCH - update partial entity
        DELETE - delete entity
   ```
https://resthooks.io/docs/guides/url

Reply:
updated